doc/env_vars.md

```diff
@@ -119,6 +119,11 @@ _NOTE:_ `LRSQL_STMT_RETRY_LIMIT` and `LRSQL_STMT_RETRY_BUDGET` are used to mitig
 | `LRSQL_KEY_ENABLE_SELFIE`         | `keyEnableSelfie`        | Boolean, whether or not to enable self-signed cert generation.                                                                                                                                                                                         | `true`                                                                                     |
 | `LRSQL_JWT_EXP_TIME`              | `jwtExpTime`             | The amount of time, in seconds, after a JWT is created when it expires. Since JWTs are not revocable, **this this time should be short** (i.e. one hour or less).                                                                                      | `3600` (one hour)                                                                          |
 | `LRSQL_JWT_EXP_LEEWAY`            | `jwtExpLeeway`           | The amount of time, in seconds, before or after the expiration instant when a JWT should still count as un-expired. Used to compensate for clock desync. Applied to both LRS and OIDC tokens.                                                          | `1` (one second)                                                                           |
+| `LRSQL_JWT_NO_VAL`                | `jwtNoVal`               | (**DANGEROUS!**) This flag removes JWT Token Validation and simply accepts token claims as configured by the associated variables below. It is extemely unlikely that you need this as it is for very specific proxy-overwrite authentication scenarios, and it poses a serious threat to system security if enabled.    | `false` |
+| `LRSQL_JWT_NO_VAL_UNAME`          | `jwtNoValUname`          | (**DANGEROUS!** See LRSQL_JWT_NO_VAL) This variable configures which claim key to use for the username when token validation is turned off.    | Not Set |
+| `LRSQL_JWT_NO_VAL_ISSUER`         | `jwtNoValIssuer`         | (**DANGEROUS!** See LRSQL_JWT_NO_VAL) This variable configures which claim key to use for the issuer when token validation is turned off.    | Not Set |
+| `LRSQL_JWT_NO_VAL_ROLE_KEY`       | `jwtNoValRoleKey`        | (**DANGEROUS!** See LRSQL_JWT_NO_VAL) This variable configures which claim key to look in for the role when token validation is turned off.    | Not Set |
+| `LRSQL_JWT_NO_VAL_ROLE`           | `jwtNoValRole`           | (**DANGEROUS!** See LRSQL_JWT_NO_VAL) This variable configures what role must be present in the key above when token validation is turned off.    | Not Set |
```

Comment:
"LRSQL_JWT_NO_VAL" should be wrapped in backticks.

Reply:
https://github.com/yetanalytics/lrsql/pull/324/commits/563b7a7298f8fda83ec087e46a5f47726589bda8